scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.49.2] - 2020-12-04
+### Added
+- Add resource request and limit to init-container in pod, from @scrosby
+- Add ability to parse single Job, update Job->Builder, from @brianbao
```

Comment:
We can remove this since this is only for the scheduler, not for the client library

Reply:
done